extensions/amp-story/1.0/amp-story.js

```diff
@@ -710,6 +735,10 @@ export class AmpStory extends AMP.BaseElement {
       true /** callToInitialize */
     );
 
+    this.storeService_.subscribe(StateProperty.SUBSCRIPTIONS_STATE, () => {
+      this.subscriptionsStatePromise_.resolve();
```

Comment:
shouldnt we navigate the user to content here?

Reply:
When the subscription state resolves, the callback would call the switchTo to the same target again. In that switchTo call, it would navigate the user to the locked page, and it would then show content if granted and show paywall if not granted.

Reply:
What if we go from `BLOCKED` to `GRANTED`?